sdk/communication/communication-identity/test/public/communicationIdentityClient.spec.ts

```diff
@@ -49,6 +49,14 @@ matrix([[true, false]], async (useAad: boolean) => {
       assert.isString(user.communicationUserId);
     });
 
+    it("successfully creates and gets a user with customId", async () => {
```

Comment:
Let's also add a test to validate that 2x creates with the same customId result in the same user id